src/context/useLoggedInUser.tsx

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from "@tanstack/react-query";
+import { useSettings } from "./useSettings";
+import { fetchIdentities } from "api/auth-identities";
+import { queryKeys } from "util/queryKeys";
+
+export const useIdentities = () => {
+  return useQuery({
+    queryKey: [queryKeys.identities],
+    queryFn: fetchIdentities,
+  });
+};
+
+export const useLoggedInUser = () => {
+  const { data: settings, isLoading, error } = useSettings();
+  const { data: identities = [] } = useIdentities();
+  let loggedInUserName = settings?.auth_user_name; //Temp val
+  //   const [loggedInUserName, setLoggedInUsername] = useState(
+  //     settings?.auth_user_name,
+  //   );
+
+  for (const client of identities) {
+    if (loggedInUserName == client.id) {
+      //   setLoggedInUsername(client.name);
+      loggedInUserName = client.name;
+    }
+  }
+
+  return {
+    loggedInUserName, //this should only return the username cross referenced by the auth username
+    auth_method: settings?.auth_user_method, //If connected via TLS, then display "Connected via TLS, else render username"
```

Comment:
let's keep the code style consistent. Return `authMethod` instead of `auth_method`. In general, snake casing should be used to extract data from server response payload or in types that represent server-side data attributes.